### ISSUE TITLE
Expose API port on docker

### ISSUE THREAD

**Issue by vrde** [bigchaindb/bigchaindb#142]
When running BigchainDB in Docker, the API port is not exposed, hence it's impossible to connect to the web API.

**Comment by thedoctor** [comment]
I'm sure you got this, but I ran into this issue and the fix involves telling docker-compose to forward the port, and also opening up address to which the web server binds – but you obviously don't want to bind to  `0.0.0.0` if running natively.
```
> git diff

──────────────────────────────────────────────────────────────────────────────────────────────
 --- a/bigchaindb/processes.py
 +++ b/bigchaindb/processes.py
──────────────────────────────────────────────────────────────────────────────────────────────
@@ -84,6 +84,5 @@ class Processes(object):

         # start the web api
         webapi = server.create_app()
-         p_webapi = mp.Process(name='webapi', target=webapi.run, kwargs={'host': 'localhost'})
+         p_webapi = mp.Process(name='webapi', target=webapi.run, kwargs={'host': '0.0.0.0'})
         p_webapi.start()


──────────────────────────────────────────────────────────────────────────────────────────────
 --- a/docker-compose.yml
 +++ b/docker-compose.yml
──────────────────────────────────────────────────────────────────────────────────────────────
@@ -24,6 +24,8 @@ bigchaindb:
     - ~/.bigchaindb_docker:/root/.bigchaindb_docker
   links:
     - rethinkdb
   ports:
+     - "5000:5000"
   environment:
     BIGCHAIN_DATABASE_HOST: rethinkdb
     BIGCHAINDB_CONFIG_PATH: /root/.bigchaindb_docker/config
```

**Comment by r-marques** [comment]
Also change the port in apiary

**Comment by vrde** [comment]
closed in #224
